rskj-core/src/test/java/co/rsk/peg/BridgeSupportSvpTest.java

```diff
@@ -60,7 +60,11 @@ class BridgeSupportSvpTest {
     private static final BridgeConstants bridgeMainNetConstants = BridgeMainNetConstants.getInstance();
     private static final NetworkParameters btcMainnetParams = bridgeMainNetConstants.getBtcParams();
     private static final FederationConstants federationMainNetConstants = bridgeMainNetConstants.getFederationConstants();
-
+    private static final List<Coin> svpFundTxOutpointsValues = Arrays.asList(
+        Coin.valueOf(1_000_000),
+        Coin.valueOf(500_000),
+        Coin.valueOf(300_000)
+    );
```

Comment:
```suggestion
    private static final List<Coin> svpFundTxOutpointsValues = Arrays.asList(
        Coin.COIN,
        bridgeMainNetConstants.getMinimumPeginTxValue(allActivations),
        bridgeMainNetConstants.getMinimumPegoutTxValue()
    );
```

I would prefer to use our values from mainnet constants.